test-result-summary-client/src/Dashboard/Widgets/Graph/IdleMicrobenchmark.jsx

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react';
```

Comment:
@sophiaxu0424 Once we are done with the clean up items such as combining benchmark name and variant. We need to work on https://github.com/AdoptOpenJDK/openjdk-test-tools/issues/118 with priority to avoid more redundant code.

Reply:
Think you've mistakenly commented in this PR @piyush286

Reply:
I have added #118 to the Code Cleanup project which has been created by agreement with you, the PMA team will do by end of March (to clean up the technical debt that has been building up by the PMA team).